test/Tester/LocalhostSiloTests.cs

```diff
@@ -88,7 +88,7 @@ public async Task LocalhostClusterTest()
             }
             finally
             {
-                var cancelled = new CancellationTokenSource();
+                using var cancelled = new CancellationTokenSource();
```

Comment:
```suggestion
                var cancelled = new CancellationTokenSource();
```

This (existing) code looks wrong those calls to the async variant of `Utils.SafeExecute` should probably be awaited